src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,86 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (x: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  perPage,
+  total,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const pageNumbers: number[] = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i++) {
```

Comment:
move this logic to func `getPageNumbers` and don't use cycle, better array methods